server/models/User.js

```diff
@@ -1,29 +1,69 @@
 const mongoose = require("mongoose");
 const bcrypt = require("bcryptjs");
 
+const bookingSchema = new mongoose.Schema({
```

Comment:
We don't need a new model here, we have a model for requests which is identical to this one (Please refer to the models/controllers within the respective folders to find the same)